dapr/clients/grpc/client.py

```diff
@@ -284,6 +284,61 @@ def publish_event(
             headers=call.initial_metadata(),
             trailers=call.trailing_metadata())
 
+    def get_state(
+            self,
+            store_name: str,
+            key: str,
+            metadata: Optional[MetadataTuple] = ()) -> DaprResponse:
+        if len(store_name) == 0 or len(store_name.strip()) == 0:
+            raise ValueError("State store name cannot be empty")
+        req = api_v1.GetStateRequest(store_name=store_name, key=key, metadata=metadata)
+        response, call = self._stub.GetState.with_call(req)
+        return DaprResponse(
```

Comment:
get_state response must have data. please define `StateResponse` by referring to https://github.com/dapr/python-sdk/blob/c592371ba38f056487be519ccf8d6c1a42a75737/dapr/clients/grpc/_response.py#L196